core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -1407,7 +1408,7 @@ class LogManager(logDirs: Seq[File],
       }
     } finally {
       if (cleaner != null) {
-        cleaner.resumeCleaning(deletableLogs.map(_._1))
+        cleaner.resumeCleaning(deletableLogs.map(_._1).toList.asJava)
```

Comment:
It's unfortunate we have to convert either the output of `pauseCleaningForNonCompactedPartitions()` or `currentLogs` from Java to Scala collections to build `deletableLogs` to then convert it to Java just here. Can we use a Java collection throughout?